Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 import "./Header.css";
 
-const HeaderTitle = ({ title }) => {
+const HeaderTitle = ({ title, time }) => {
   return (
     <div className="HeaderTitle">
       <p>
-        {title} <span>@lambda 12:00pm</span>
+        {title} <span>@lambda {time}</span>
       </p>
     </div>
   );
```

Comment:
Timi great job on the assingment. You succesfully completed the Social Card MVP. Your code is clean and You are properly splitting up the code into multiple components. There are only minor issues like removing the borders and putting some border-radius on the top part of the image.
Your git workflow is fluid and You did great  job on this assingment 👍